Directory.Build.props

```diff
@@ -34,9 +34,8 @@
         Use fixed version of analyzers.
     -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="8.0.0-preview1.23165.1" PrivateAssets="all" />
```

Comment:
> Remove reference to Microsoft.CodeAnalysis.NetAnalyzers (use the stable version by default)
You should not remove this package because of .NET Framework do not have analyzers by default, should you?

Reply:
According to https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers, it is included by virtue of `<EnableNETAnalyzers>true</EnableNETAnalyzers>` on line 21

Reply:
But I think you still need to install Nuget. You can check it?

Reply:
The packages README says

> You do not need to manually install this NuGet package to your project if you are using .NET5 SDK or later

We are using .NET8 SDK. An example of a CA analyzer on net462 target:

![image](https://github.com/sshnet/SSH.NET/assets/5132141/820c7252-832f-46f4-9143-fa0da144bab5)